contracts/contracts/core/Oracle.sol

```diff
@@ -152,27 +152,27 @@ contract Oracle is OracleStorage, IOracle, Ownable2StepUpgradeable, UUPSUpgradea
 
     /**
      * @dev Internal function to process a commitment, either slashing or rewarding based on the commitment's state.
-     * @param commitmentIndex The id of the commitment to be processed.
+     * @param commitmentDigest The hash of the commitment to be processed.
      * @param isSlash Determines if the commitment should be slashed or rewarded.
      * @param residualBidPercentAfterDecay The residual bid percent after decay.
      */
     function _processCommitment(
-        bytes32 commitmentIndex,
+        bytes32 commitmentDigest,
```

Comment:
Why is this changed? The caller of this function still uses commitmentIndex. Does this have to be renamed? We use the commitmentIndex when this is called from Oracle.

Reply:
Changing this primarily to keep a consistent naming convention

Reply:
https://github.com/primev/mev-commit/blob/main/oracle/pkg/updater/updater.go#L407

What I meant is this argument seems to be commitmentIndex. I dont think this is commitmentDigest.

Reply:
Yeah, you're right. It's a hash of the digest+sig. https://github.com/primev/mev-commit/blob/50f6efefd0836cf7e790d59ded277a37e338196d/contracts/contracts/core/PreconfManager.sol#L624